apps/frontend/src/layout/partials/Navbar.tsx

```diff
@@ -1,17 +1,35 @@
 import {
```

Comment:
@AfeezGL The `Navbar` is becoming a mammoth and I believe most of the logic here can be refactored and restructured. Can you coordinate with @afolasope to do this? I already explain what needs to be done. For example, there's no reason we have the create project dialog here. It should be a separate component. I can also see some repeated `DropdownMenu` components here that we can easily refactor.

Reply:
@jdamilola We'll look into that.